### PR TITLE
Create simpson.cpp

### PR DESCRIPTION
**Pull request by kichloo**
Integration using Simpson's 1/3 rule

**Fixes issue:*
Integration using Simpson's rule

**Changes:**
Added the code according to clang format

### CODE REVIEW

File: simpson.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    float value = 0;
+    for (int i = 0; i <= n; ++i) {
+        if (i == 0 || i == n)
+            value += y[i];
+        else if (i % 2 != 0)
+            value += 4 * y[i];
+        else
+            value += 2 * y[i];
+    }
+    value = value * (h / 3);
```
Comment (arnavb) [inline simpson.cpp]:
```suggestion
    value *= (h / 3);
```

File: simpson.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+float create(int x)
+{
+    return log(x);
+}
+
+int simpson(int x1, int x2)
+{
+
+    float h = (x2 - x1) / 6;
+
```
Comment (arnavb) [inline simpson.cpp]:
```suggestion
```

File: simpson.cpp [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+#include <math.h>
+
+float create(int x)
+{
+    return log(x);
+}
+
+int simpson(int x1, int x2)
+{
+
```
Comment (arnavb) [inline simpson.cpp]:
```suggestion
```

File: simpson.cpp [hunk 4]
```diff
@@ -0,0 +1,41 @@
+#include <iostream.h>
+#include <math.h>
```
Comment (arnavb) [inline simpson.cpp]: Use cmath.

File: simpson.cpp [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    return value;
+}
+
+int main()
+{
+    cout << "Enter the lower limit";
+    cin >> x1;
+    cout << "Enter the upper limit";
+    cin >> x2;
+    cout << simpson(x1, x2);
```
Comment (arnavb) [inline simpson.cpp]:
```suggestion
    cout << simpson(x1, x2) << '\n';
```

**Comment by arnavb** [comment]
When implementing my suggestions, do not create a new pull request.

**Comment by kichloo** [comment]
@arnavb Done the required changes.

File: simpson.cpp [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    return value;
+}
+
+int main()
+{
+    cout << "Enter the lower limit";
+    cin >> x1;
+    cout << "Enter the upper limit";
+    cin >> x2;
+    cout << simpson(x1, x2)<<"\n";
```
Comment (arnavb) [inline simpson.cpp]:
```suggestion
    cout << simpson(x1, x2)<< "\n";
```
Also move your file to [this folder](https://github.com/OpenGenus/cosmos/tree/master/code/mathematical_algorithms/src/simpsons_rule) and call it `simpsons_rule.cpp`.
Also move the file to a new directory

**Comment by kichloo** [comment]
@arnavb I could not move the file to the required destination, so I created PR in it.

**Pull request closed without merging**